tokenizers/javascript/parser.js

```diff
@@ -54,22 +54,40 @@ function extractForFolder(dir, identifiersOnly, outpath, baseDir) {
     fs.writeFileSync(outpath, zlib.gzipSync(gzipped));
 }
 
-// For each project (folder in folder)
-var base_dir = '/mnt/c/Users/t-mialla/Downloads/data';
-var all_user_folders = fs.readdirSync(base_dir);
-for (var i in all_user_folders) {
-    var user_folder = path.join(base_dir, all_user_folders[i]);
-    if (!fs.lstatSync(user_folder).isDirectory()) {
-        continue;
-    }
-    all_project_folders = fs.readdirSync(user_folder);
-    for (var j in all_project_folders) {        
-        var project_folder = path.join(user_folder, all_project_folders[j]);        
-        if (!fs.statSync(project_folder).isDirectory()) continue;
+if (process.argv.length != 4) {
+    console.error(`Usage: ${process.argv[0]} PROJECTS_FOLDER OUTPUT_FOLDER`);
```

Comment:
Is this maybe 
```suggestion
    console.error(`Usage: ${process.argv[1]} PROJECTS_FOLDER OUTPUT_FOLDER`);
```
I assume that `process.argv[0]` is commonly `node`?

Reply:
Good catch! I think this message should print both argv[0] & argv[1], i.e. `node parser.js PROJECTS_FOLDER OUTPUT_FOLDER`. I'll make a commit